Functions/FirebaseFunctions/FIRFunctions.m

```diff
@@ -131,7 +131,13 @@ - (instancetype)initWithProjectID:(NSString *)projectID
 }
 
 - (void)useLocalhost {
-  [self useFunctionsEmulatorOrigin:@"http://localhost:5005"];
+  [self useEmulatorWithHost:@"http://localhost" port:5005];
```

Comment:
Host should not contain `http://` ... unless I am missing something about this Functions API

Reply:
Removed.